impl/src/main/java/org/kuali/rice/kim/service/impl/IdentityManagementServiceImpl.java

```diff
@@ -429,12 +428,6 @@ protected void addIsAuthorizedToCache( String key, boolean authorized ) {
 		isAuthorizedCache.put( key, new MaxAgeSoftReference<Boolean>( permissionCacheMaxAgeSeconds, authorized ) );
 	}
 
```

Comment:
Removing getAuthenticatedPrincipalName method was the only changed in this class.